source/coding/git.rst

```diff
@@ -1,12 +1,150 @@
 Git
-============================
+===
```

Comment:
If you want some ideas on command to add and explanations, here's the presentation I started https://docs.google.com/presentation/d/1z0gsgM2Of3TIBSmJUVNPJJR6UkWZEEE_OBVYixOT8iA/edit?usp=sharing

It's a shareable link you can also put as reference on the page.